res/css/views/elements/_AccessibleButton.scss

```diff
@@ -39,6 +39,7 @@ limitations under the License.
         justify-content: center;
         font-size: $font-14px;
         border: none; // override default <button /> styles
+        word-break: keep-all;
```

Comment:
can we get a comment to say this fixes a language issue? I'm also generally a bit concerned about including it at this level, but we'll see what breaks from it (we have a lot of magic flexboxes and such).

Reply:
Addressed: https://github.com/matrix-org/matrix-react-sdk/pull/8212/commits/29ab4073e125bc1e76e76672ed20ed7a58fa4ac6

I'll continue to keep an eye on buttons in Japanese UI- this kind of global change will not be able to be conducted after the number of users from China/Japan/Korea increases.